src/software/SfM/main_ComputeCCTagStructureFromKnownPoses.cpp

```diff
@@ -19,6 +19,180 @@ using namespace openMVG;
 using namespace openMVG::sfm;
 namespace bfs = boost::filesystem;
 
+void alignAndScaleStructure(
+    SfM_Data & sfmData,
+    const IndexT nCCTags,
+    const IndexT indexOrigin)
+{
+  const std::size_t nLandmarks = sfmData.GetLandmarks().size();
+  Mat3X vX(3,nLandmarks);
+  std::size_t i=0;
```

Comment:
i'd rename it differently, like `idx` or something like that, as u are using it for all the `for` loops as well. Not that it is a problem but better avoid the same name